contracts/sol6/SanityRatesMaxGasPrice.sol

```diff
@@ -0,0 +1,59 @@
+pragma solidity 0.6.6;
+
+import "./utils/Utils5.sol";
+import "./utils/Withdrawable3.sol";
+
+
+contract SanityRatesMaxGasPrice is Withdrawable3, Utils5 {
+    mapping(address => uint256) public tokenRate;
```

Comment:
am thinking now

we can actually create a structure

struct SanityData{
    uint224 tokenRate;
    uint32 reasonableDiffInBps;
}

and so we have one mapping. and we save one storage read per token.

Reply:
Done